internal/flags/client.go

```diff
@@ -19,6 +19,7 @@ type UpdateInput struct {
 
 type Client interface {
 	Create(ctx context.Context, accessToken, baseURI, name, key, projKey string) ([]byte, error)
+	Read(ctx context.Context, accessToken, baseURI, key, projKey, envKey string) ([]byte, error)
```

Comment:
Could you make this `Get` like the subcommand?

Reply:
We can do that. Was trying to follow CRUD originally. But it got confusing at points

Reply:
Done